og.install

```diff
@@ -74,3 +74,10 @@ function og_schema() {
 
   return $schema;
 }
+
+/**
+ * Implements hook_install().
+ */
+function og_install() {
+  \Drupal\og\Og::groupResolverHandler()->updateConfigStorage();
+}
```

Comment:
shouldn't be needed, as the plugins should not be configurable. The only configuration is their weight and status (enabled/ disabled)